### ISSUE TITLE
Not possible to set up Permissions-Policy security header due to restriction in JSON schema imposed by ZEIT dependency

### ISSUE THREAD

**Issue by roelnoort** [vercel/serve#649]
Hi,

I am using `serve.json` to set up security headers.
Now that the `Feature-Policy` has changed to `Permissions-Policy` *and* has a new syntax this poses an issue.

The header I need to add is:
```
{ 
  "headers": [ 
    { 
      "source" : "*", 
      "headers": [
        ...
        {
          "key": "Permissions-Policy",
          "value": "geolocation=(), microphone=(), camera=()"
        }
      ]
    }
  ]
}
```

However, this results in the following error:
```
INFO: Discovered configuration in `serve.json`
ERROR: The configuration you provided is wrong:
should match pattern "^[a-zA-Z0-9_!#$%&'*+.;/:, =^`|~-]+$"
{"pattern":"^[a-zA-Z0-9_!#$%&'*+.;/:, =^`|~-]+$"}
```

This error is thrown by `serve.js` because the call to `ajv.compile(schema)` returns an error.

The `schema` is set up like:
```
const schema = require('@zeit/schemas/deployment/config-static');
```

I checked the source of this repository and indeed found that the `value` property of the json I'm trying to insert gets validated like this:
```
value: {
	type: 'string',
	minLength: 1,
	maxLength: 2048,
	pattern: "^[a-zA-Z0-9_!#$%&'*+.;/:, =^`|~-]+$"
}
```

In other words, it shows the pattern used to validate the string. This fails because the open and close bracket symbols ( and ) are not included in the pattern.

I'm not too sure what the @zeit/schemas repository is. But I think it either needs to be updated here on in that repository (in case that is meant for similar purposes).

If you agree with the above, could this be implemented in serve or could you help contacting @zeit on your behalf?

Thanks, Roel

**Comment by GuillermoBlasco** [comment]
Same issue here. It would be great to see that PR merged <3

**Comment by paulshryock** [comment]
Let's do this. 👊

**Comment by trhinehart-godaddy** [comment]
Also running into this issue but with `<` and `>` characters when trying to set `Link` header. https://www.geeksforgeeks.org/http-headers-link/
